### PR TITLE
User api

### LINKED ISSUE #17: Add API Routes for Users

### PR DESCRIPTION
**Pull request by wjrhee**
revised per comments.
closes #17

### CODE REVIEW

File: server/app/routes/user/user.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    })
+})
+
+router.get('/', function(req, res, next){
+
+  return User.findAll({
+    attributes: ['first_name', 'last_name','email']
+  })
+    .then(function(users){
+      // send back only first name, last name, and email for now.  Needs to be secured to only allow admins to access this data
```
Comment (Abazhenov) [inline server/app/routes/user/user.js]: if you res.json(users) you will return the whole thing, just do res.json({firstname: xxx lastname: xxx : whatever: xxx}) I'm not sure of the exact database syntax

File: server/app/routes/user/user.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+const router = require('express').Router();
+const db = require('../../../db');
+const User = db.model('user');
+
+// Need to secure routes to only be accessible by admins
+
+router.get('/:id', function(req, res, next){
+  return User.findById(req.params.id)
+    .then(function(user){
```
Comment (Abazhenov) [inline server/app/routes/user/user.js]: do same thing you did for findall here

**Comment by wjrhee** [comment]
updated findById router

**Comment by wjrhee** [comment]
@Abazhenov adjusted findById to return specific fields

**Pull request merged**